keybase/platform_windows.go

```diff
@@ -62,6 +66,63 @@ func (c context) BeforeUpdatePrompt(update updater.Update, options updater.Updat
 	return nil
 }
 
+type regUninstallGetter func(string, Log) bool
+
+// It turns out a Wix bundle .exe supports "/layout", which among other things
+// generates a log of what it would do. We can parse this for Dokan product code variables,
+// which will be in the registry if the same version is already present.
+func CheckCanBeSilent(path string, log Log, regFunc regUninstallGetter) bool {
+	tempName, err := util.WriteTempFile("keybaseInstallLayout", []byte{}, 0700)
```

Comment:
oh this is prefix... 

I've been meaning to have file extension support on that method... but for now you should put a dash or period... like `KeybaseInstallLayout.` or `KeybaseInstallLayout-` for the prefix.

path := TempPath("",)